docs/src/userguide/io/index.rst

```diff
@@ -155,6 +155,13 @@ Only supported in batch mode.
 
 Klio supports BigQuery columns as event inputs. More information about configuring BigQuery table inputs can be found in the :ref:`event-config-bigquery` event config section.
 
+Avro
+^^^^
+
```

Comment:
I would mention that Avro is only supported in batch mode (like how it says for [BQ](https://klio--151.org.readthedocs.build/en/151/userguide/io/index.html#google-bigquery))

Reply:
Yeah i have this in the line right below "Both local and GCS avro files are supported in batch read and write modes."

But I'll give the batch only callout it's own line to make the callout very obvious!